src/nrnpython/nb_defs.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
```

Comment:
Nanobind already as a file called `nd_defs.h`, why do we want reuse the name?

Reply:
I only copy the file and remove what is uneeded

Reply:
We cannot use the original file?

Reply:
Discussed offline. Summary: not a fight worth picking right now.